### DISCUSSION TITLE
What's the proxy configuration

### DISCUSSION THREAD

**Discussion by caibirdme** [vectordotdev/vector#14554] [Q&A] [answered]
I found [proxy](https://vector.dev/docs/reference/configuration/sinks/clickhouse/#proxy) config in sources like nginx prometheus ... and sinks like clickhouse. But I don't really understand what does it mean?
When should we use the proxy?

**Comment by jszwedko** [comment, marked as answer]
Hi @caibirdme !

The `proxy` configuration can be used if you need to route requests through an [HTTP proxy](https://en.wikipedia.org/wiki/Proxy_server) to the configured endpoints.
